aries_cloudagent/protocols/coordinate_mediation/v1_0/manager.py

```diff
@@ -144,10 +146,12 @@ def updated_to_keylist_updated(updated: RouteUpdated):
         updated = map(updated_to_keylist_updated, updated)
         return KeylistUpdateResponse(updated=updated)
 
-    async def get_keylist(self, record: MediationRecord) -> Sequence[RouteRecord]:
+    async def get_keylist(self, record: MediationRecord) -> Keylist:
```

Comment:
Tests for this method will need to change to reflect the different return value.

Reply:
Actually, `create_keylist_query_response` should be doing the creation of the keylist message already. It does the mapping from route record to message.

Reply:
I think I understand. Let me update this.